app/services/envirotech/toolkit_scraper.rb

```diff
@@ -0,0 +1,56 @@
+module Envirotech
+  class ToolkitScraper
+    def initialize
+      agent.get('https://new.export.gov/envirotech/toolkit')
+    rescue
```

Comment:
Usually best to be explicit about the type of errors you wish to rescue. As it is here, you're silencing any error that arises, not only ones that you expect.